selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -74,10 +68,18 @@ void WifiManager::setup() {
   requestScan();
 }
 
+void WifiManager::start() {
+  timer.start(5000);
+  refreshNetworks();
+}
```

Comment:
@deanlee Active scan takes a second, are we leaving enough time until we get the APs? This could be why it doesn't show a lot of APs right after opening the networking view

Reply:
oh I see we have a signal set up to refresh again later. I suppose it's best to leave this here in case that breaks some how